pfcpiface/config_test.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"io/fs"
+	"io/ioutil"
+	"testing"
+)
+
+func mustWriteStringToDisk(s string, path string) {
+	err := ioutil.WriteFile(path, []byte(s), fs.ModePerm)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func TestParseJSON(t *testing.T) {
+	t.Run("config is preserved", func(t *testing.T) {
+		s := `{
```

Comment:
Committing and testing a file verbatim is not without downsides.
This one is identical to the aether-helm-charts _now_, but wont forever.
But, it forces us to update the test when the config structure changes ("I should tell OPs") and serves as an example.

Open to both keeping and removing it.

Reply:
I think it's better to keep it - due to the rationale you provided.

Reply:
I wonder if we could test on the included `conf/upf.json`.

Reply:
Makes sense too